Anytype/Sources/PresentationLayer/Modules/Membership/Views/MembershipLegalButton.swift

```diff
@@ -0,0 +1,34 @@
+import SwiftUI
+
+struct MembershipLegalButton: View {
+    let text: String
+    let onTap: () -> ()
+    
+    var body: some View {
+        Button {
+            onTap()
+        } label: {
+            HStack(spacing: 0) {
+                AnytypeText(text, style: .bodyRegular, color: .Text.primary)
+                    .lineLimit(1)
+                Spacer()
+                IconView(icon: .asset(.X18.help))
```

Comment:
asset x18, but frame is 16? 🤔

Reply:
It is temporary asset. waiting for designers to add right one to the design system